### PR TITLE
hotfix: modify uikit config payload format

### PR DESCRIPTION
**Pull request by AhyoungRyu**
### Description Of Changes
Currently storybook is throwing a runtime error due to the wrong json format access which is caused by my prev PR.

### CODE REVIEW

File: src/lib/UIKitConfigProvider/index.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ const UIKitConfigProvider = ({ storage, children, localConfigs }: UIKitConfigPro
     try {
       if (storage == null) {
         const payload = await sdk.getUIKitConfiguration();
-        setRemoteConfigs(snakeToCamel(payload.json));
+        setRemoteConfigs(snakeToCamel(payload.json).configuration.uikitConfigurations);
```
Comment (AhyoungRyu) [inline src/lib/UIKitConfigProvider/index.tsx:39]: I shouldn't have removed `uikitConfigurations` this one but I guess it's removed by mistake 😅

Comment (AhyoungRyu) [reply src/lib/UIKitConfigProvider/index.tsx:39]: Also, the json data format has been changed in the latest version of sendbird/chat; `configuration` is added in the root level of payload.

**Review by sravan-s — APPROVED** [review]
THnx

**Review by sravan-s — APPROVED** [review]
THnx

**Pull request merged**